kolibri/core/content/management/commands/importchannel.py

```diff
@@ -120,21 +120,22 @@ def _start_file_transfer(self, filetransfer, channel_id, dest):
                         filetransfer.cancel()
                         break
                     progress_update(len(chunk), progress_extra_data)
-                try:
-                    import_channel_by_id(channel_id, self.is_cancelled)
-                except channel_import.ImportCancelError:
-                    # This will only occur if is_cancelled is True.
-                    pass
                 if self.is_cancelled():
                     try:
                         os.remove(dest)
-                    except IOError as e:
+                    except OSError as e:
                         logger.error(
```

Comment:
I wonder if it's going to make more sense here to display the message as info instead of an error message.
in the current code, when cancelling channel import (database download), although in the UI everything looks great, errors will be shown in the log file, which may be confusing to users or developers who look at the log file to debug other issues.
```
ERROR    Tried to remove /Users/lingyiwang/.kolibri/content/databases/a9b25ac9814742c883ce1b0579448337.sqlite3, but exception [Errno 2] No such file or directory: '/Users/lingyiwang/.kolibri/content/databases/a9b25ac9814742c883ce1b0579448337.sqlite3' occurred.
```

Reply:
Yeah - it's intended behaviour, so we shouldn't error.